src/engine/cuecontrol.cpp

```diff
@@ -14,9 +14,10 @@
 #include "vinylcontrol/defs_vinylcontrol.h"
 
 static const double CUE_MODE_MIXXX = 0.0;
-static const double CUE_MODE_PIONEER = 1.0;
-static const double CUE_MODE_DENON = 2.0;
-static const double CUE_MODE_NUMARK = 3.0;
+static const double CUE_MODE_MIXXX_NO_BLINK = 1.0;
```

Comment:
Wouldn't it have to be listed last in the drop down in the preferences? It would be weird to have it there instead of next to Mixxx mode (or Pioneer mode if it is renamed).